test-app/package.json

```diff
@@ -22,14 +22,14 @@
     "lint:js:fix": "eslint . --fix",
     "lint:types": "glint",
     "start": "ember serve",
-    "test": "npm-run-all lint \"test:!(watch)\"",
+    "test": "pnpm lint && pnpm test:ember",
```

Comment:
For some reason after this move, it wasn't recognizing `npm-run-all`.  Since we aren't running these in parallel, `&&` is essentially the same thing I think reading https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/npm-run-all.md.

Reply:
`npm-run-all` comes from some older version of the addon blueprint I think. We use `concurrently` by now, see https://github.com/CrowdStrike/ember-headless-form/blob/main/test-app/package.json#L15